### ISSUE TITLE
conversion for the fs25

### ISSUE THREAD

**Issue by syxvcb** [modelleicher/FS22_realismAddon_gearbox#28]
hello mitico, will there be any hope for a conversion for the fs25? without this mod it's really a pain.....

**Comment by KrS14** [comment]
Agreed, I won't even think of buying FS25 unless this is updated for it.

**Comment by Mmike1984** [comment]
I really hope there comes a fs25 version. Shifting now is bad, pls be our hero. 😅
